oaknational/Oak-Web-Application
# PUPIL-536 - Feat/404 sensitive lessons

## Description

Music year: 2012

- List of changes
query lessons with isSensitive field
404 any lesson which is sensitive

## Issue(s)

Fixes #

## How to test

1. Go to [abortion page](https://deploy-preview-2331--oak-web-application.netlify.thenational.academy/pupils/programmes/rshe-pshe-secondary-ks4-l/units/intimate-sexual-relationships-pregnancy-and-parenting-ff99/lessons/abortion-6xh38d)
2. [online world vs real life page](https://deploy-preview-2331--oak-web-application.netlify.thenational.academy/pupils/lessons/how-the-online-world-is-different-to-real-life-c8vk4d)
3. Click on \_\_\_\_\_\_
4. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
